### DISCUSSION TITLE
Can you disable VRR?

### DISCUSSION THREAD

**Discussion by rlf037** [MonitorControl/MonitorControl#981] [Ideas]
Basically, VRR is broken in Monterey on some monitors. It causes flickering issues. Turning it off fixes the issue but the problem is after the monitor awakes from sleep it defaults back to VRR on even after you've turned it to a fixed RR. Can you make VRR disabled permanently?

**Comment by the0neyouseek** [comment]
Hi @rlf037 currently the app has no effect on VRR, it's only here to control brightness and volume. I don't think it's in the scope of this app to do that as it's quite a specific display issue.

I think you can disable VRR directly in your macos display settings forcing a fixed RR, this display settings should normally be saved for the monitor so that they're applied everytime you connect to it.

Depending on your monitor, it may also have a setting to disable VRR/Freesync on it directly.

**Reply by rlf037** [reply]
Hi, thanks for the reply.

I was seeking it as a possible feature add, given you have intimate knowledge of how to interact with displays on a macOS environment. You can disable VRR but the issue is everytime the monitor awakes from sleep, VRR turns back on automatically. The disabling option doesn't persist so I was wondering if your program could control it. It is called MonitorControl after all !

I believe this is strictly a Monterey issue and it is VERY common.

Unfortunately, my monitor doesn't offer the option to disable VRR on it directly.

**Reply by waydabber** [reply]
Hi @rlf037 - on my LG display when I use USB-C, I can't disable Freesync and it indeed messes with Monterey. If I change to Displayport, I have an option to disable VRR and strangely it also affects USB-C.

Anyway, in BetterDummy I just implemented detecting VRR modes https://github.com/waydabber/BetterDummy/issues/422 - I could technically add a feature there like "Auto switch to corresponding non-VRR mode when VRR mode detected" or something like this. I am not sure how common this need is though.

**Reply by waydabber** [reply]
I added an enhancement for this https://github.com/waydabber/BetterDummy/issues/423 as this sometimes bothered me as well until I figured out how to disable Freesync finally.
